test/specs/elements/Header/_Header-test.js

```diff
@@ -0,0 +1,13 @@
+import _Header from 'src/elements/Header/_Header'
+import * as common from 'test/specs/commonTests'
+
+describe('_Header', () => {
+  common.propKeyOnlyToClassName(_Header, 'left', { className: 'aligned' })
+  common.propKeyOnlyToClassName(_Header, 'center', { className: 'aligned' })
+  common.propKeyOnlyToClassName(_Header, 'right', { className: 'aligned' })
```

Comment:
The last parameter to common tests is `requiredProps`.  These are props that are required for the component to render without errors.  So, any `propTypes` that are marked `isRequired`.

The 3rd parameter is not required here for 2 reasons:
1. There are required props for a _Header
1. The common test sets `className`, overriding `className` in `requiredProps`

Hope that helps clear up the common tests' `requiredProps` option.  LMK if not and I'll get some more information on this, perhaps in a doc somewhere.